tests/functional/splinter_test.c

```diff
@@ -2593,20 +2606,19 @@ splinter_test(int argc, char *argv[])
    data_config       *data_cfg;
    clockcache_config *cache_cfg =
       TYPED_ARRAY_MALLOC(hid, cache_cfg, num_tables);
-   test_exec_config test_exec_cfg;
-   ZERO_STRUCT(test_exec_cfg);
 
    rc = test_parse_args_n(splinter_cfg,
                           &data_cfg,
                           &io_cfg,
                           &al_cfg,
                           cache_cfg,
                           &log_cfg,
-                          &test_exec_cfg,
+                          &test_exec_cfg, // Extract test-execution params here
```

Comment:
This is a weird comment.

Reply:
(I thought it was self-explanatory.  This parsing routine disburses diff config params to diff locations. The comment is trying to guide the reader that this class of "test-execution" parameters are parsed into this struct. Would like to retain it.)

Reply:
It's not clear to me why the reader especially cares about the `test_exec_cfg` vs any of the other configs. The format draws a lot of attention to it and it doesn't seem that important to me.

I feel as though a comment above the function saying that this parses the command line into the various configs would be more useful, since then it's obvious from context.

Reply:
I took it out. Adding an extra comment just at this invocation may require adding a similar comment in all other places where it's called.  The called function's prolog (I see) already documents that the configs are distributed to various other config sub-structs.